docs/user_guide/data_existing_cat.rst

```diff
@@ -12,7 +12,87 @@ The summary per dataset contains links to the online source and available litera
 The ``deltares_data`` catalog is only available within the Deltares network. However a selection of this data for a the
 Piave basin (Northern Italy) is available online in the ``artifact_data`` archive and will be used if no data catalog is provided.
 Local or other datasets can also be included by extending the data catalog with new yaml :ref:`data catalog files <data_yaml>`.
-We plan to provide more data catalogs with open data sources in the (near) future. See the data catalog `changelog <https://github.com/Deltares/hydromt/blob/main/data/catalogs/changelog.rst>`_ for recent updates on the pre-defined catalogs.
+We plan to provide more data catalogs with open data sources in the (near) future.
+See the data catalog `changelog <https://github.com/Deltares/hydromt/blob/main/data/catalogs/changelog.rst>`_ for recent updates on the pre-defined catalogs.
 
+Using a predefined catalog
+--------------------------
+
+From CLI
+~~~~~~~~
+
+To use a predefined catalog, you can specify the catalog name with the ``--dd`` or ``--data_catalog`` option when running a HydroMT command.
```

Comment:
I think this is incorrect. The current code in `cli/main.py` specifies that `--dd` is for the detlares data catalog specifically not other predefined catalogs, those are loaded by default if I recall correctly. Can you check this, and update if necessary?

Reply:
Could also be that we need to rework the cli option, but in that case it should also move to v1

Reply:
Yes, you're right Sam. I changed it.